script.js

```diff
@@ -1,4 +1,553 @@
+        return "Hard luck. You have busted. Let's wait for the dealer to draw its remaining cards.<br> Click <b>'Submit'</b> to to see the results."; // then change mode to dealer hits
+      } else {
+        // if player's score is less than 21 after card 4, ask player to hit or stand.
+        currentScore_player = calcTotalScore(playerDrawnCards);
+        var message_displayCards4 = `Player's Hand:<br> Card 1: ${playerDrawnCards[0].name} of ${playerDrawnCards[0].suit} <br> Card 2: ${playerDrawnCards[1].name} of ${playerDrawnCards[1].suit}<br> Card 3: ${playerDrawnCards[2].name} of ${playerDrawnCards[2].suit}<br> Card 4: ${playerDrawnCards[3].name} of ${playerDrawnCards[3].suit}<br> Player's score: ${currentScore_player} <br><br> Dealer's Hand:<br> Card 1: *Hidden* <br> Card 2: ${dealerDrawnCards[1].name} of ${dealerDrawnCards[1].suit}<br><hr>`;
+        return `${message_displayCards4} Player, do you want to hit or stand? <br> Type <b>'hit'</b> or <b>'stand'</b> and click <b>'Submit'</b>.`;
+      }
+      // if player chooses to stand, record final score, gameMode changes to dealer hits.
+    } else if (input == "stand") {
+      finalScore_player = calcTotalScore(playerDrawnCards);
+      gameMode = DEALER_HITS;
+      return "You have chosen to stand. Let's wait for the dealer to draw its remaining cards.<br> Click <b>'Submit'</b> to to see the results.";
+    }
+  } else if (gameMode == PLAYER_HITS && playerDrawnCards.length == 4) {
+    if (input != "hit" && input != "stand") {
+      return "Oops that is invalid. Please type <b>'hit'</b> or <b>'stand'</b> and click <b>'Submit'</b>. ";
+    } else if (input == "hit") {
+      // Player chooses to draw card 5.
+      playerDrawnCards.push(shuffledDeck.pop()); // Player Card 5
+      // if player busted, dealer draws card 3.
+      if (calcTotalScore(playerDrawnCards) > 21) {
+        playerHasBusted = true;
+        console.log(playerHasBusted);
+        finalScore_player = calcTotalScore(playerDrawnCards);
+        gameMode = DEALER_HITS;
+        return "Hard luck. You have busted. Let's wait for the dealer to draw its remaining cards.<br> Click <b>'Submit'</b> to to see the results."; // then change mode to dealer hits
+      } else {
+        // if player's score is less than 21 after card 5, player stands.
+        finalScore_player = calcTotalScore(playerDrawnCards);
+        gameMode == DEALER_HITS;
+        var message_displayCards5 = `Player's Hand:<br> Card 1: ${playerDrawnCards[0].name} of ${playerDrawnCards[0].suit} <br> Card 2: ${playerDrawnCards[1].name} of ${playerDrawnCards[1].suit}<br> Card 3: ${playerDrawnCards[2].name} of ${playerDrawnCards[2].suit}<br> Card 4: ${playerDrawnCards[3].name} of ${playerDrawnCards[3].suit}<br> Card 5: ${playerDrawnCards[4].name} of ${playerDrawnCards[4].suit}<br> Player's score: ${finalScore_player} <br><br> Dealer's Hand:<br> Card 1: *Hidden* <br> Card 2: ${dealerDrawnCards[1].name} of ${dealerDrawnCards[1].suit}<br><hr>`;
```

Comment:
Whenever you find yourself hardcoding, it can definitely be done in a better and more efficient way. Imagine if the player was allowed to draw 20 cards (hypothetically), would you hard code all 20 cards?
Try using a for loop for this.